software/elasticsearch/egg-elasticsearch.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-07-24T15:19:46-05:00",
+    "name": "Elasticsearch",
+    "author": "dominic@stilma.net",
+    "description": null,
+    "features": null,
+    "images": [
+        "quay.io\/pterodactyl\/core:java-11"
```

Comment:
Why use java 11?

Reply:
It requires java, however i noticed it has an in-built java as well.